linter/linter.go

```diff
@@ -1499,6 +1501,30 @@ func (l *Linter) lintFunctionArguments(calledFn functionMeta, ctx *context.Conte
 			calledFn.meta, calledFn.name,
 			len(fn.Arguments), len(calledFn.arguments),
 		).Match(FUNCTION_ARGUMENTS).Ref(fn.Reference))
+	} else if argTypes[0] == types.StringListType {
+		// Variadic arguments linting, at least one argument must be provided and must be a StringType
+		if len(calledFn.arguments) == 0 {
+			err := &LintError{
+				Severity: ERROR,
+				Token:    calledFn.token,
+				Message: fmt.Sprintf(
+					"function %s variadic arguments must be provided at least one argument",
```

Comment:
nit: I think grammar is not correct here. maybe

`"function %s  requires at least one argument"`